plugins/metagen.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: /plugin.njk
+title: Metagen
+readmeUrl: https://raw.githubusercontent.com/tannerdolby/eleventy-plugin-metagen/master/README.md
+categories:
+  - utility
+maintainer:
+  name: tannerdolby
+  url: https://tannerdolby.com
+githubUrl: https://github.com/tannerdolby/eleventy-plugin-metagen
+description: "An Eleventy shortcode that generates metadata containing: Open
+  Graph, Twitter card, generic meta tags, CSS, JS, custom tags, and a canonical
+  link."
+---
+# eleventy-plugin-metagen
+An Eleventy [shortcode](https://www.11ty.dev/docs/shortcodes/) that generates document metadata containing: Open Graph, Twitter card, generic meta tags, CSS, JS, custom tags, and a canonical link.
+
+## Installation
+Install the plugin from [npm](https://www.npmjs.com/package/eleventy-plugin-metagen):
+
+```
+npm install eleventy-plugin-metagen
+```
+
+Add it to your [Eleventy Config](https://www.11ty.dev/docs/config/) file:
+
+```js
+const metagen = require('eleventy-plugin-metagen');
+
+module.exports = (eleventyConfig) => {
+    eleventyConfig.addPlugin(metagen);
+};
+```
+
+## What does it do?
+The plugin turns [11ty shortcodes](https://www.11ty.dev/docs/shortcodes/) like this:
+
+```nunjucks
```

Comment:
The `metagen` example usage in this code snippet seems to be what's breaking the build after looking at the [deploy logs](https://app.netlify.com/sites/plug11ty/deploys/62b3f124121c7e000b2b1e89). Usually I have to wrap these snippets in `{% raw %} ... {% endraw %}` tags when dealing with this on my own site.

```
$ npx eleventy
9:51:07 PM: [11ty] Problem writing Eleventy templates: (more in DEBUG output)
9:51:07 PM: [11ty] 1. Having trouble rendering liquid template ./plugins/metagen.md (via TemplateContentRenderError)
9:51:07 PM: [11ty] 2. tag "metagen" not found, file:./plugins/metagen.md, line:25, col:1 (via ParseError)
9:51:07 PM: [11ty] 3. tag "metagen" not found (via AssertionError)
9:51:07 PM: [11ty]
9:51:07 PM: [11ty] Original error stack trace: AssertionError: tag "metagen" not found
```